.jenkinsci/artifacts.groovy

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env groovy
+
+def uploadArtifacts(filePaths, uploadPath, artifactServers=['artifact.soramitsu.co.jp']) {
+  def baseUploadPath = 'files'
+  def filePathsConverted = []
+  agentType = sh(script: 'uname', returnStdout: true).trim()
+  uploadPath = baseUploadPath + uploadPath
+  filePaths.each {
+    fp = sh(script: "for i in \$(ls -d ${it}); do FILES=\${i},\${FILES};done;echo \$FILES", returnStdout: true).trim()
```

Comment:
Prefer `'` instead of `"`, so there is no escapes

Reply:
same below

Reply:
`${it}` here refers to an element of `for ... each` loop and should not be escaped. However, escaped variables refer to the shell variables and not defined previously in Groovy script. Usage of single quotes would work for intra-shell vars (like `FILES` here) but `${it}` will be interpreted like a literal string and not a Groovy variable.